utils/dd_to_csv.py

```diff
@@ -13,7 +12,7 @@
 
 def parse_parameter_values_from_file(
     path: Path,
-) -> Tuple[Dict[str, List], Dict[str, set]]:
+) -> tuple[dict[str, list], dict[str, set]]:
```

Comment:
Am curious why the lowercase type annotations are preferable? (I'm happy either way)

Reply:
The lowercase hints allow the use of built-in types.  It avoid having to `from typing import Dict, List, ...` and lets you just hint with `dict, list` etc.  Just syntactic sugar.  Ruff updated all these automatically.